tools/proto-compiler/src/cmd/compile.rs

```diff
@@ -183,35 +171,22 @@ impl CompileCmd {
     }
```

Comment:
Should `.extern_path(".ics23", "::ics23")` be `.extern_path(".ics23", "crate::cosmos::ics23")` or `.extern_path(".ics23", "::cosmos_sdk_proto::cosmos:ics23")` ?

Reply:
Ended with 

```rust
            .extern_path(".cosmos.ics23.v1", "::ics23")
```

though

```
            .extern_path(".cosmos.ics23.v1", "crate::ics23")
```

could work as well